### ISSUE TITLE
importing mixcr dataset via tcR

### ISSUE THREAD

**Issue by zuta-osa77** [imminfo/tcr#79]
hi all, 

I'm new to this: I've run mixcr/1.8.1 on my RNA-seq dataset using the partialAssemble workflow described on the main page and would like to create the analog of the `twa` list object in R which represents the Cloneset. I have several questions: 

1) I'm guessing I need to run exportClones with a couple of specified fields in order to use 

`parse.mitcr`

directly on it. Is this correct? If so, would anybody have the exact call to exportClones which would allow the output to be directly input into a Cloneset object in R? I don't know in other words how `twa` or `twb` were generated or cherrypicked from exportClones output

2) Along the same lines, how do I combine all the exportClones output from mixcr so that I can represent my entire dataset in a Cloneset, e.g. all four subjects are in `twa`

Basically, if there was a concrete example of how to go from N clns files (from mixcr assemble) to N exportClones files with the right fields picked out to be directly input into tcR, and then make a N x 1 list containing the entire dataset in a Cloneset object, that would be great!

Looks like a great set of routines and visualization scripts, so I can't wait to use it. 

all the best,
zo

**Comment by vadimnazarov** [comment]
Hi,

1) You should use `parse.mixcr` in order to parse the MiXCR files. Yes, first you run exportClones, and after that you parse the output files to R with `parse.mixcr`. I think you could just run `mixcr exportClones clones.clns clones.txt`, i.e., without any parameters, to get the suitable for tcR output tables.

2) You just need to put all your clns files to the same folder, and after that run `your_list = parse.folder(path_to_your_folder, "mixcr")`. Run `?parse.folder` for more info.

I hope this helps. If not please message me here or by email.

**Comment by zuta-osa77** [comment]
hey thanks Vadim

**Comment by vadimnazarov** [comment]
No problems, please let me know if it works for you or not, so I would be able to help you / close this issue.

**Comment by zuta-osa77** [comment]
Yes this worked out, thanks again for your help. I have another related question concerning filtering this imported dataset -- would you prefer in the interests of good organization that I open a new issue, ask you this in private, or just ask here?

**Comment by vadimnazarov** [comment]
Nice! Regarding another question - please, either open a new issue or send me an email, which way is more comfortable for you.
